chore(gen): docs content review (#3586)
scaleway/docs-content: dedibox-console/classic-hosting/how-to/retrieve-headers-of-email.mdx

The header of an email contains essential information about the path and origins of the email before it arrives in your mailbox. It is comparable to the postmark and routing information on a physical envelope sent through the mail.

## Why email headers are important

Email headers can provide crucial information about the origin of an email, the servers it passed through, and various technical details that may help in troubleshooting or verifying the authenticity of the email.
Email headers are often required when reporting spam, investigating delivery issues, or diagnosing potential security threats. In some cases, email headers may contain information required by technical support to diagnose issues or verify the authenticity of an email. The method for retrieving this information varies depending on the email client you use.

## Retrieving email headers

### How to retrieve email headers in Thunderbird

1. Select the email in your inbox that you need to investigate.
2. In the preview pane, click **Others**, then select **View Source**. A new window will open displaying the email's source code, including its headers.
3. Copy the relevant header information from this window.

> ### How to retrieve email eeaders in Gmail

Reviewer: ```suggestion
### How to retrieve email headers in Gmail
```

1. Log in to your Gmail account.

> 2. Open the email whose headers you need.

Reviewer: ```suggestion
2. Open the email containing the desired headers.
```

3. Click the **downward arrow** in the upper-right corner of the email (next to the reply button).
4. Select **Show original** from the dropdown menu. The headers will be displayed in a new window.
5. Copy the header information as needed.

### How to retrieve email headers in Outlook

1. Navigate to your list of messages and double-click the email you wish to view. This will open the message in a new window.
2. Click on **File** > **Properties**. A new window will appear showing the email properties.

> 3. The email headers can be found in the "Internet headers" section. Copy the headers from this section.

Reviewer: ```suggestion
3. Copy the headers from the "Internet headers" section.
```

### How to retrieve email headers in Outlook Web Mail

1. Log in to your Microsoft account.
2. Open the email for which you want to retrieve the headers.

> 3. Click the **downward arrow** in the email's top right. A menu will appear.

Reviewer: ```suggestion
3. Click the **downward arrow** in the email's top right corner. A menu appears.
```
